locale/gd.js

```diff
@@ -0,0 +1,85 @@
+//! moment.js locale configuration
```

Comment:
these are a result of running grunt release - we need these in the right places so that moment can find them when used in production